src/test/suite/fixtures/recorded/decorations/carveLineHarp.yml

```diff
@@ -0,0 +1,38 @@
+languageId: plaintext
+command:
+  version: 4
+  spokenForm: carve line harp
+  action: {name: cutToClipboard}
+  targets:
+    - type: primitive
+      modifiers:
+        - type: containingScope
+          scopeType: {type: line}
+      mark: {type: decoratedSymbol, symbolColor: default, character: h}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |2
+        hello world
+  selections:
+    - anchor: {line: 1, character: 0}
+      active: {line: 1, character: 0}
+  marks:
+    default.h:
+      start: {line: 0, character: 4}
+      end: {line: 0, character: 9}
+finalState:
+  documentContents: ""
+  selections:
+    - anchor: {line: 0, character: 0}
+      active: {line: 0, character: 0}
+  clipboard: "hello world"
+ide:
+  flashedRanges:
+    - style: referenced
+      range:
+        type: character
+        start: {line: 0, character: 4}
+        end: {line: 0, character: 15}
+    - style: pendingDelete
+      range: {type: line, start: 0, end: 0}
```

Comment:
notice the new flash style for `"carve line"`: we highlight the whole line as deleted, but show just the content as referenced